### ISSUE TITLE
Mithril's copy should be emotional

### ISSUE THREAD

**Issue by pygy** [MithrilJS/mithril.js#1860]
@JAForbes [suggested](https://gitter.im/mithriljs/mithril.js?at=591db5929f4f4ab05bd8ede0) these copy updates:

> Mithril's copy should be emotional
> 
> > Everything you know about JS just became even more powerful
> 
> Or with some shade
> 
> > Mithril solves problems you care about, instead of re-inventing javascript in html
> 
> Instead of presenting graphs, be direct, be quoteable.
> 
> > Mithril is faster than react, smaller than vue, does more, with less.
> 
> This isn't enough:
> 
> > Mithril is used by companies like Vimeo and Nike, and open source platforms like Lichess.
> 
> You have to say why, and it has be more than dry facts
> 
> > Mithril's abstractions scale to projects of any size, that's why companies like Vimeo and Nike choose mithril.
> 
> This is close
> 
> > Mithril supports browsers all the way back to IE9, no polyfills required.
> 
> Should be
> 
> > No polyfills, no build system, just build stuff in less time and less code

**Comment by isiahmeadows** [comment]
Agreed. And sadly, every time I've tried suggesting this (I've tried several times already, both here and in Gitter), I've managed to somehow get pushback for it. 😞

As for why I agree:

1. Programmers are people, and people are irrational. Emotional arguments are *way* better at persuading people than logical ones.
2. *We* are pretty clearly emotionally supportive of this library. But our page looks like we're all a bunch of Vulcan clones. I mean, if I'm looking at frameworks and libraries, the first thing that comes to mind is "What is it?", not "How large is it?" or "How fast is it?".

-----

(I mean, not like [this](https://github.com/MithrilJS/mithril.js/issues/1599) [hasn't](https://github.com/MithrilJS/mithril.js/issues/1603) [been](https://github.com/MithrilJS/mithril.js/issues/1446) [discussed](https://github.com/MithrilJS/mithril.js/issues/1026) [before](https://github.com/MithrilJS/mithril.js/issues/433) numerous times...sorry if I come across a little harsh here... 😖)

**Comment by pdfernhout** [comment]
A couple of other ideas:  

"Burned out from wrestling with leaky abstractions that are difficult to debug? Try making your app with Mithril and bring joy back to your coding."

Or:

 "Weave joy throughout your code with Mithril."

**Comment by timbotron** [comment]
"If you like your code being as light as a feather, but as hard as dragon scales, choose Mithril."

**Comment by NetOperatorWibby** [comment]
Can this go into Planned/In-Progress?

**Comment by NetLancer** [comment]
Plenty of nice suggestions above, great thanx to devGuys !
I personally got attracted to Mithril when first came across by its existing descriptions/specs + comparison with notorious giants. And surely i still love the framework.
But it seems to me - it lacks real world apps (of use) made with mithril as there are made with vue/react. (like vue-crud or in combo with some backend framework/cms for beautiful robust apps).
I'm in process of getting a better knowledge of the frameworks but generally we need guys who know the product well, love it and are creative enough to USE it as a well-made building material in some industry

**Comment by casajarm** [comment]
My 2 cents on emotion...an appeal that would work on me is one that focuses on how NOT frustrating it is. It's easy to convince myself to use a library or framework and then after investing a lot of effort I start to find cracks and things that I can't seem to grok. Ultimately, to use any bit of code I have to live with decisions the authors made. At least in the case of Mithril I can tell that this group has been very conscientious about those decisions.
